pkg/trace/obfuscate/sql.go

```diff
@@ -312,18 +313,21 @@ func (oq *ObfuscatedQuery) Cost() int64 {
 
 // attemptObfuscation attempts to obfuscate the SQL query loaded into the tokenizer, using the
 // given set of filters.
-func attemptObfuscation(tokenizer *SQLTokenizer) (*ObfuscatedQuery, error) {
+func attemptObfuscation(tokenizer *SQLTokenizer, options *config.SQLObfuscationConfig) (*ObfuscatedQuery, error) {
+	var quantizeTableNames bool
+	if options != nil {
+		quantizeTableNames = options.QuantizeSQLTables
+	}
```

Comment:
Do we want to also consider the value of `features.Has("quantize_sql_tables")` too?

Reply:
Addressed here: https://github.com/DataDog/datadog-agent/pull/8556#discussion_r666560122